### ISSUE TITLE
New read-only member role

### ISSUE THREAD

**Issue by alexkiro** [getsentry/sentry#4956]
I think a good feature for private projects would be to have the ability to create members that can only view/search over issues but not actually act on them. (assigning and resolving)

There doesn't seem to be such a role at the moment: https://docs.sentry.io/learn/membership/

**Comment by dcramer** [comment]
Agreed. I think we have scopes well off enough that its possible to create a role, but I'm not entirely sure thats true.

e.g. right now the 'member' role is:

```
         'scopes': set([
            'event:read', 'event:write', 'event:delete', 'project:releases',
            'project:read', 'org:read', 'member:read', 'team:read',
        ]),
```

In theory if just created a role without project:releases and event:write that'd prevent mutation on the majority of things. There's potentially some things that just check project:read when you're acting upon yourself, but those are either safe, or we'd just need to make tweaks.

**Comment by jgriffith23** [comment]
Checked for an existing pull request on this, and I don't see one. @scotchka and I will help.

**Comment by BYK** [comment]
I think this is closed by #5097
